sycl/ReleaseNotes.md

```diff
@@ -16,6 +16,699 @@
 
 - Did this and that ... intel/llvm#pr
 
+# Release notes Mar'25
+
+Release notes for commit range
+[b0212c37b2](https://github.com/intel/llvm/commit/b0212c37b230d9dd3bb129df9f4ecc417b92ad8)
+...
+[b23d69e2c3](https://github.com/intel/llvm/commit/b23d69e2c3fda1d69351137991897c96bf6a586d)
+
+## New Features
+
+### Runtime compilation of SYCL code
+
+- [`sycl_ext_oneapi_kernel_compiler`](https://github.com/intel/llvm/blob/b23d69e2c3fda1d69351137991897c96bf6a586d/sycl/doc/extensions/experimental/sycl_ext_oneapi_kernel_compiler.asciidoc)
+  extension specification was updated to accept `sycl` as source language, thus
+  providing functionality similar to
+  [NVRTC](https://docs.nvidia.com/cuda/nvrtc/). intel/llvm#11985,
+  intel/llvm#17446
+- Initial support for this feature was implemented. intel/llvm#16132,
+  intel/llvm#16222, intel/llvm#16132, intel/llvm#17640, intel/llvm#17356,
+  intel/llvm#16565, intel/llvm#17383, intel/llvm#17447, intel/llvm#17307,
+  intel/llvm#17373, intel/llvm#17331, intel/llvm#17329, intel/llvm#17266,
+  intel/llvm#17032, intel/llvm#16823, intel/llvm#16702, intel/llvm#16638,
+  intel/llvm#16316, intel/llvm#17359, intel/llvm#16485, intel/llvm#16821
+- Known issues and limitations are documented
+  [in the extension specification](https://github.com/intel/llvm/blob/b23d69e2c3fda1d69351137991897c96bf6a586d/sycl/doc/extensions/experimental/sycl_ext_oneapi_kernel_compiler.asciidoc#non-normative-implementation-notes-for-dpc). intel/llvm#17307,
+  intel/llvm#17459
+
+### SYCL graphs
+
+- Introduced and implemented
```

Comment:
```suggestion
- Implemented support for the
```
Nit since we didn't write the extension just implemented support for it in graphs.